x-pack/plugins/canvas/public/components/element_types/element_types.js

```diff
@@ -32,38 +45,63 @@ export const ElementTypes = ({ elements, onClick, search, setSearch }) => {
           textAlign="left"
           image={image}
           title={displayName}
-          description={help}
+          description={
+            <Fragment>
+              {help}
+              <br />
+              <TagList tags={tags} tagType={tagType} />
+            </Fragment>
+          }
```

Comment:
We can make the badges appear in the same place (bottom) of each card to tidy things up layout-wise. Just use the `footer` prop on EuiCard like so:

```
...
          description={
            <Fragment>
              {help}
            </Fragment>
          }
          footer={
            <TagList tags={tags} tagType={tagType} />            
          }
...
```

Reply:
Good call! The spacing looks much nicer.

![screen shot 2019-01-15 at 3 41 37 pm](https://user-images.githubusercontent.com/1697105/51214768-1d013180-18dc-11e9-8b1d-38e78ca31633.png)